WAL/MANIFEST PR 요청입니다

isu-kim opened the pull request:
## Update
- #35 PR 에서 지적해주신 가독성 부분을 수정했습니다.
- 또한 제가 공부한 내용과 추가적으로 적을 내용들을 추가해서 문서를 작성했습니다.

늦게 올리게 되어서 정말 죄송합니다.
추가적으로 수정해야하는 사항이 생기시면 말씀해주시면 반영해서 다시 PR 하도록 하겠습니다.

isu-kim committed:
Update wal.md

- doc: add documentation for WAL/MANIFEST

isu-kim committed:
Update wal.md

- doc: change minor order issue

isu-kim committed:
Update wal.md

doc: resize image size

min-guk commented:
네, 고생하셨습니다.

1. 
우선 내용이 너무 많아서, WAL하고 Manifest를 분리해주셨으면합니다.
Manifest는 목차에 따로 빼두었으니, 그 쪽에 작성해주세요.
WAL도 내용이 많다면 문서를 분리해주세요. 

2. 
올려주신 그림 파일에 빨간색 밑줄이 쳐져있습니다.
PPT에서 그림으로 저장을 활용하시거나, 사전에 추가 기능을 사용해주세요.

3. 
log reader는 ppt를 캡쳐해서 사용하셨는데, 학생분들께서 바쁘신 건 알지만 조금 더 이쁘게 정리해주셨으면 합니다.
제가 기간도 많이 지났고 바쁘신 걸 알아서 넘어가려고 했는데, 다른 학생들과의 형평성도 맞지 않는 것 같아서 다시 한번 정리부탁드립니다.
그림만 따로 저장해서 올려주시고, 제목은 마크다운 문법을 활용해주세요.

isu-kim commented:
> 우선 내용이 너무 많아서, WAL하고 Manifest를 분리해주셨으면합니다.
Manifest는 목차에 따로 빼두었으니, 그 쪽에 작성해주세요.
WAL도 내용이 많다면 문서를 분리해주세요.

이건 WAL, MANIFEST 두가지 파일로 분리하도록 하겠습니다.

> 올려주신 그림 파일에 빨간색 밑줄이 쳐져있습니다.
PPT에서 그림으로 저장을 활용하시거나, 사전에 추가 기능을 사용해주세요.

놓친 부분이라 깜빡하고 그냥 넘긴 것 같습니다. 해당 내용 수정해서 다시 올리도록 하겠습니다. 지적 감사합니다.

> log reader는 ppt를 캡쳐해서 사용하셨는데, 학생분들께서 바쁘신 건 알지만 조금 더 이쁘게 정리해주셨으면 합니다.

해당 내용 담당 학생에게 전달 후 다시 업데이트해서 올리도록 하겠습니다.

감사합니다!

isu-kim committed:
Fix document for #37's comment

- doc: split WAL and MANIFESt
- doc: replace assets for better quality (no red lines)

isu-kim committed:
Update wal.md

doc: change picture

isu-kim committed:
Update wal.md

Update log::reader::ReadRecord

Co-Authored-By: SayOny <83440202+SayOny@users.noreply.github.com>

isu-kim commented:
> log reader는 ppt를 캡쳐해서 사용하셨는데, 학생분들께서 바쁘신 건 알지만 조금 더 이쁘게 정리해주셨으면 합니다.

해당 내용을 담당하는 팀원의 내용을 추가해서 다시 PR 을 진행했습니다. 이외 지적해주신 두가지 사항은 수정했습니다. 시간 나시면 PR 한번 봐주시고 수정 사항 있으시면 말씀해주시면 감사하겠습니다. 감사합니다

min-guk commented:
네, 고생하셨습니다.
